library/common/config/config.cc

```diff
@@ -39,6 +39,8 @@ const std::string config_header = R"(
 - &dns_fail_max_interval 10s
 - &dns_query_timeout 25s
 - &dns_preresolve_hostnames []
+- &h2_connection_keepalive_idle_interval 100000s
```

Comment:
Probably should be 0 here just so it's disabled in vanilla config.

Reply:
unfortunately the proto constraint to at least 1ms.